Mixpanel/MPABTestDesignerConnection.m

```diff
@@ -15,11 +15,13 @@
 #import "MPDesignerSessionCollection.h"
 #import "MPLogger.h"
 #import "MPSwizzler.h"
+#import "MPResources.h"
 
 NSString * const kSessionVariantKey = @"session_variant";
+static const int connectivityIndicatorSize = 50;
```

Comment:
One more minor thing, can we prefix this or do all caps so it's obvious it is a constant, something like

kConnectivityIndicatorSize or CONNECTIVITY_INDICATOR_SIZE

Why did you opt for a constant instead of checking the size of the image at run time?